app/src/main/java/com/paxus/pay/poslinkui/demo/entry/text/AVSFragment.java

```diff
@@ -92,9 +92,9 @@ protected void loadView(View rootView) {
             editTextAddr.setFilters(new InputFilter[]{new InputFilter.LengthFilter(maxLengthZip)});
         }
         if(zipText){
-            editTextAddr.setInputType(android.text.InputType.TYPE_CLASS_TEXT);
-        }else {
             editTextAddr.setInputType(android.text.InputType.TYPE_CLASS_NUMBER);
+        }else {
+            editTextAddr.setInputType(android.text.InputType.TYPE_CLASS_TEXT);
```

Comment:
Double confirm variable name. zipText should control input type of ZipCode. Not Address.

Reply:
This dependency of editTextAddr on zipText has been here from before. The InputType logic was reversed.

Reply:
Old codes not always RIGHT.  Address should accept text. Zip should accept number only or accept text.